apps/admin/middlewares/extract.ts

```diff
@@ -0,0 +1,25 @@
+import { NextApiRequest } from "next";
+import { getSession } from "next-auth/react";
+import {
+  fetchActionToAction,
+  ReactAdminFetchActions,
+  Permissions,
+} from "ra-data-simple-prisma";
+import { authProvider } from "../providers/authProvider";
+
+export const extractAccessObjFromReq = async (req: NextApiRequest) => {
+  const session = await getSession({ req });
```

Comment:
does it not work with getServerSession?

Reply:
not tested yet, but this is the old code, which should work

Reply:
fixed, changed to getServerSession